atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -191,30 +193,17 @@ def _get_receivers_list(self):
                            r.status_code, r.text)
             raise RuntimeError('PDC returned non-200 status code (%s), see referenced build log' %
                                r.status_code)
-        # TODO: open an RFE for PDC to allow querying by dist_git_branch, now we have to filter
-        #  the correct result manually
-        found = []
-        for result in r.json()['results']:
-            if result['dist_git_branch'] == git_branch:
-                found.append(result)
-
-        if len(found) != 1:
-            self.log.error('expected 1 PDC component, found %s: %s',
-                           len(found), found)
-            raise RuntimeError('Expected to find exactly 1 PDC component, found %s, '
-                               'see referenced build log' % len(found))
 
-        send_to = []
-        contacts = found[0].get('contacts', [])
-        for contact in contacts:
-            # I'm not sure, but it seems that there can be more people with the role
-            if contact['contact_role'] == self.pdc_contact_role:
-                send_to.append(contact['email'])
+        contacts = r.json()
 
-        if len(send_to) == 0:
+        if contacts['count'] == 0:
             self.log.error('no %s role for the component', self.pdc_contact_role)
             raise RuntimeError('no %s role for the component' % self.pdc_contact_role)
 
+        send_to = []
+        for contact in contacts['results']:
+            send_to.append(contact['contact']['email'])
+
```

Comment:
I guess I'm just used to dealing with CUPS ;-) but I've always found it useful to perform easy checks like that on the client end rather than always trusting the server.

OK, happy with it as it is.